shpc/main/modules/base.py

```diff
@@ -387,22 +393,27 @@ def install(
         container_dir = self.container.container_dir(subfolder)
 
         # Are we also installing to a named view?
-        if view is None and not disable_view:
-            view = self.settings.default_view
+        view_names = []
```

Comment:
Hmm.. so I think allowing two views here is confusing. It should be either or (how it was before). because If we allow multiple the next logical question is "why can't we provide more than one as an argument?" and I don't want to support that!

Reply:
It's still allowed here because this function is what `shpc view install` calls. In other words, this function here needs to  do i) core installation, ii) default view installation, iii) custom view installation.

My opinion is that this method shouldn't be dealing with views at all: just doing core installation. The reason I didn't move view installation away is that it relies on `View` objects that are only instantiated in this class. Moving everything will make the diff larger.

Note that for the reinstall feature, I will need support _somewhere_ for multiple views.